west.yml

```diff
@@ -147,7 +149,8 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 84157212bb4e45454869ee07822605f8f00ad551
+      revision: fake_802154_54l10
```

Comment:
Just leaving a comment for you to not forget about it :)

Reply:
changed to real PR